src/bin.rs

```diff
@@ -22,6 +22,8 @@ fn main() {
     opts.optflag("q", "quiet", "Hide regular progress output");
     opts.optmulti("e", "exclude", "Don't scan directories or files with that filename (wildcards are not supported)", "<exact filename>");
     opts.optflag("", "json", "Display results as JSON");
+    opts.optflag("r", "reflink", "Strict reflinking (copy-on-write) instead of hardlinking - WILL FAIL IF unsupported");
+    opts.optflag("f", "reflink-or-hardlink", "Try reflinks first, fallback to hardlinks if reflinks are not supported");
```

Comment:
`f` is usually used for `force` flags. Please use `c` for `reflink-or-hardlink` and `C` for `reflink`